packages/docs/pages/_app.tsx

```diff
@@ -46,38 +47,42 @@ export default class MyApp extends App {
             }
           `}
         </style>
-        <ThemeProvider theme={theme}>
-          <>
-            <GlobalStyles />
-            {router.query.noNav ? (
-              <Component {...pageProps} />
-            ) : (
+        <UIDReset>
```

Comment:
Are we going to have to add these to our microapps now? 🤔Should we create a HOC for everything that's essential:
```tsx
const Wrapper = ({ children, theme }) => (
  <UIDReset>
    <UIDFork>
      <ThemeProvider theme={theme}>
        <GlobalStyles />
        {children}
      </ThemeProvider>
    </UIDFork>
  </UIDReset>
);
```
Just a 💭

Reply:
We don't need to add them to our microapps. These are only to provide better SSR support. We can create a wrapper in the future when we need it.